src/pybind/mgr/tox.ini

```diff
@@ -4,18 +4,27 @@ envlist =
     mypy,
     test,
     fix
+    flake8
 skipsdist = true
 requires = cython
 
 [flake8]
 max-line-length = 100
+ignore =
+    E501,
+    F241,
+    F401,
+    F811,
+    F841,
+    W503,
```

Comment:
Ignoring these for now since they are more involved to review/fix. Will follow-up with changes in a separate PR.